src/modules/text.js

```diff
@@ -0,0 +1,45 @@
+const textWithoutArguments = (element) => {
+    if (element.length === undefined) {                 
+         return element.textContent;
+     } else {
+         let textString = '';
+         for (let item of element) {
+             textString += ` ${item.textContent}`
+         }
+         return textString;
```

Comment:
You can convert `element` to Array and use `map` method to make the code more readable , so your `else` block can look like 
`return element.map(item => item.textContent).join(' ')`